app/src/main/java/com/sunasterisk/moviedb_51/data/model/Cast.kt

```diff
@@ -0,0 +1,12 @@
+package com.sunasterisk.moviedb_51.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Cast(
```

Comment:
```suggestion
data class CastAttribute(
```
Chuyển class Casts gộp vào đây luôn e